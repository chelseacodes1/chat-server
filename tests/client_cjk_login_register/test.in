SAY 茶馆 也许以后还有缘相见。
CREATE 茶馆
LOGIN ユーザー1号 猎人壹
REGISTER ユーザー1号 猎人壹
LOGIN ユーザー1号 猎
LOGIN ユーザー1号 猎人壹
LOGIN ユーザー1号 猎人壹