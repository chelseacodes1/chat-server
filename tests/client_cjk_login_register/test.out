Error: user not logged in, cannot send message
Error: User not logged in, cannot create channel
RESULT LOGIN 0
RESULT REGISTER 1
RESULT LOGIN 0
RESULT LOGIN 1
RESULT LOGIN 0
